tcti/sockets.c

```diff
@@ -104,6 +107,26 @@ InitSockets( const char *hostName,
 
     }
 
+#if defined(TSS2_USE_SO_NOSIGPIPE)
+    iResult = setsockopt(*otherSock, SOL_SOCKET, SO_NOSIGPIPE, (void*)&no_sigpipe_val, sizeof(no_sigpipe_val));
+    if (iResult)
+    {
+        SAFE_CALL( debugfunc, data, NO_PREFIX, "setting SO_NOSIGPIPE failed with error = %d\n", WSAGetLastError() );
+        closesocket( *otherSock );
+        WSACleanup();
+        return(1);
+    }
+
+    iResult = setsockopt(*tpmSock, SOL_SOCKET, SO_NOSIGPIPE, (void*)&no_sigpipe_val, sizeof(no_sigpipe_val));
+    if (iResult)
+    {
+        SAFE_CALL( debugfunc, data, NO_PREFIX, "setting SO_NOSIGPIPE failed with error = %d\n", WSAGetLastError() );
+        closesocket( *otherSock );
+        WSACleanup();
+        return(1);
```

Comment:
No need for the brackets around 1

Reply:
Fixed.  Both styles are used in the existing code in that function.